local/network.go

```diff
@@ -329,7 +352,14 @@ func loadDefaultNetworkFiles() (map[string]interface{}, []byte, []*utils.NodeKey
 }
 
 // NewDefaultConfigNNodes creates a new default network config, with an arbitrary number of nodes
-func NewDefaultConfigNNodes(binaryPath string, numNodes uint32, networkID uint32) (network.Config, error) {
+func NewDefaultConfigNNodes(
+	binaryPath string,
+	numNodes uint32,
+	networkID uint32,
+	genesisPath string,
+	upgradePath string,
+	beaconConfig map[ids.NodeID]netip.AddrPort,
```

Comment:
lets just receive a boolean, not the config

Reply:
not a part of the PR